scripts/publish_pr.py

```
@@ -92,10 +92,21 @@ def check_working_tree_clean() -> bool:
 
 
 def get_changed_files(branch: str) -> list[str]:
-    """Get list of changed files in the current branch vs its merge base with main."""
+    """Get list of changed files: either uncommitted changes or commits on the branch."""
     try:
+        # First check for uncommitted changes
         result = subprocess.run(
-            ["git", "merge-base", branch, "origin/main"],
+            ["git", "diff", "--name-only"],
```

Ruff (0.15.17)

[error] Starting a process with a partial executable path

(S607)